### PR TITLE
Avoid showing draft revert message on autosaves

### LINKED ISSUE #12147: Nagging "Post reverted to draft" bug
**Describe the bug**
Notification "Post reverted to draft" is displayed constantly whilst editing a post.

xivrrxum5x
Video with play controls: http://recordit.co/XiVrRXUM5x

**Additional context**
- Gutenberg plugin version 4.5.0

### PR DESCRIPTION
**Pull request by youknowriad**
Fix #12147

**Testing instructions**

 - Publish a post
 - Edit the post
 - Wait for autosave
 - The "reverted to draft" message should not show up

### CODE REVIEW

File: packages/editor/src/store/effects/posts.js [hunk 1]
```diff
@@ -170,10 +170,10 @@ export const requestPostUpdate = async ( action, store ) => {
  * @param {Object} store   Redux Store.
  */
 export const requestPostUpdateSuccess = ( action ) => {
-	const { previousPost, post, isAutosave, postType } = action;
+	const { previousPost, post, postType } = action;
 
 	// Autosaves are neither shown a notice nor redirected.
-	if ( isAutosave ) {
+	if ( get( action.options, [ 'isAutosave' ] ) ) {
```
Comment (gziolo) [inline packages/editor/src/store/effects/posts.js]: Yes, we changed that. Now, I remember.

**Review by gziolo — COMMENTED** [review]
I will test soonish.

**Comment by gziolo** [comment]
Unit test failed

**Comment by youknowriad** [comment]
yes, looking.

**Review by gziolo — APPROVED** [review]
I confirm it fixes the issue.

**Pull request merged**